[0m11:39:10.228024 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E667F19430>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E667F1AFC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E667F1A000>]}


============================== 11:39:10.232024 | fbb710c5-94ba-45de-971a-4857807ea12b ==============================
[0m11:39:10.232024 [info ] [MainThread]: Running with dbt=1.8.7
[0m11:39:10.232024 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\Matt\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init ai_ingestion', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:39:10.252069 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/Matt/.dbt')}
[0m11:39:10.253069 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:39:10.256083 [debug] [MainThread]: Starter project path: C:\Users\Matt\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m11:39:10.276921 [info ] [MainThread]: 
Your new dbt project "ai_ingestion" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:39:10.277922 [info ] [MainThread]: Setting up your profile.
[0m11:39:10.279922 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m11:39:10.281922 [debug] [MainThread]: Command `dbt init` failed at 11:39:10.280922 after 0.16 seconds
[0m11:39:10.281922 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E667CC9640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E6688C1D90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E66888CCE0>]}
[0m11:39:10.281922 [debug] [MainThread]: Flushing usage events
